packages/grid/x-data-grid/src/hooks/features/filter/gridFilterSelector.ts

```diff
@@ -19,6 +19,15 @@ export const gridFilterModelSelector = createSelector(
   (filterState) => filterState.filterModel,
 );
 
+/**
+ * Get the current quick filter values.
```

Comment:
```suggestion
 * Gets the current quick filter values.
```

Reply:
All the other lines are with `Get`

Reply:
That's inconsistent because all API methods use the present tense. Anyway, leave as it is and we can fix all later.